index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="/styles.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="main">
+        <header id="header">
+            <div class="logo">
+                <img src="images/tromb.png" alt="picture" id="header-imng" />
+                <nav id="nav-bar">
+                    <ul>
+                        <li><a class="nav-link" href="#features">Features</a></li>
+                        <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+                        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                    </ul>
+                </nav>
+            </div>
```

Comment:
-Please take advantage of the <main> tag and use it instead of the div element to make the code more sematic 😊, lines 12 to 24 
-Use of more sematic tags could significantly Reduce the number of div's in overall codebase